### ISSUE TITLE
GeoLite2 database not free anymore

### ISSUE THREAD

**Issue by MarcHagen** [shlinkio/shlink#596]
#### How Shlink is set-up

* Shlink Version: 1.21.0
* PHP Version: 7.3.13
* How do you serve Shlink: Self-hosted swoole
* Database engine used: MariaDB

#### Summary

The URL `http://geolite.maxmind.com/download/geoip/database/GeoLite2-City.tar.gz` doesn't exist anymore. 

More here: https://dev.maxmind.com/geoip/geoip2/geolite2/
Maybe including there package? https://maxmind.github.io/GeoIP2-php/

#### Current behavior
`bin/cli visit:update-db` results in `[ERROR] An error occurred while updating GeoLite2 database`

#### How to reproduce

If i'm incorrect please let me know but upgrading doesn't work for me right now. Very annoying

**Comment by acelaya** [comment]
Apparently, it continues being free, but they have changed how it has to be used.

I will have to investigate and find some workaround.

Thanks for letting me know about this and sorry for the inconveniences this might cause.

**Comment by MarcHagen** [comment]
Ah fair enough, for now i changed the URL according to https://forum.matomo.org/t/maxmind-is-changing-access-to-free-geolite2-databases/35439/3

**Comment by jimmybrancaccio** [comment]
@MarcHagen Where did you change it? I'm trying to install Shlink now but even after updating the URL in:

```
./config/autoload/geolite2.global.php
./vendor/shlinkio/shlink-ip-geolocation/src/GeoLite2/GeoLite2Options.php
```

the installation fails. The URL I used is:

https://web.archive.org/web/20191227182209/https://geolite.maxmind.com/download/geoip/database/GeoLite2-City.tar.gz

**Comment by MarcHagen** [comment]
Ah well i changed it to a local copy url, not the archive.org because it's officially not allowed by there terms. Sorry was not very specific about that

**Comment by acelaya** [comment]
You can use this URL in order to continue getting the latest DB file `https://download.maxmind.com/app/geoip_download?edition_id=GeoLite2-City&license_key=G4Lm0C60yJsnkdPi&suffix=tar.gz`.

I will publish a new patch release right away which will use this new URL.

**Comment by acelaya** [comment]
I have just released v1.21.1, which includes the fix.

Thank you guys for reporting the issue and providing the info to find a solution :muscle:
